report_pdf_zip_download/static/src/js/action_manager_report.esm.js

```diff
@@ -0,0 +1,54 @@
+// © 2017 Creu Blanca
+// Copyright 2024 Quartile (https://www.quartile.co)
+// License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import {download} from "@web/core/network/download";
+import {registry} from "@web/core/registry";
+import {user} from "@web/core/user";
+
+registry
+    .category("ir.actions.report handlers")
+    .add("zip_handler", async function (action, options, env) {
+        if (
```

Comment:
We can consolidate the context first?
```suggestion
        const ctx = { ...(user?.context || {}), ...(action?.context || {}) };
        if (
```